hypebot/plugins/league/game_lib.py

```diff
@@ -453,6 +480,41 @@ def GetChampionImageUrl(self, champ_name):
       return None
     return self.GetImageUrl('champion', champ.image.full)
 
+  def GetReforgedRuneMessage(self, rune_name):
+    """Gets the reforged rune."""
+    if self._reforged_runes_name_complete:
+      rune = self._reforged_runes_name_complete.GuessThing(rune_name)
+      if rune:
+        return self._ComputeReforgedRuneCard(rune)
+
+    return 'Reforged rune "{}" not found.'.format(rune_name)
+
+  def _ComputeReforgedRuneCard(self, rune):
+    # Make runes more dank.
+    rune_name = util_lib.Dankify(rune.name)
+    tooltip = util_lib.Dankify(rune.long_desc)
+    var_table = client_vars.REFORGED_RUNE_VARS.get(rune.key, {})
+    tooltip = re.sub(
+        ur'@.+?@',
+        lambda match: var_table.get(match.group(0), ur'¯\_(ツ)_/¯'), tooltip)
+    slot = self._reforged_rune_slot[rune.key]
+    path_description = '{} {}'.format(
+        rune.rune_path_name, 'Keystone'
+        if slot == 0 else 'Tier {}'.format(slot))
+
+    rune_strs = []
+
+    rune_strs.append('{}: {}'.format(rune_name, path_description))
+
+    tooltip_strs = []
+    for line in util_lib.AsText(tooltip).split(u'<br>'):
```

Comment:
Don't quote me on this, but I'm relatively sure that AsText doesn't exist anymore.